index.html

```diff
@@ -40,4 +41,143 @@ <h1 class="heading">
         </ul>
       </div>
     </section>
-    
\ No newline at end of file
+    <section class="work-section">
+      <div class="project-card">
+
+        <img src="images/photo1.png" alt="First Card">
+
+        <h2 class="head">Tonic</h2>
+        <div class="card-detail-box">
```

Comment:
- [ ] Nice job designing the project cards 💯 but the heading `Tonic` is too close to the image. It would be great to add a little margin-top to the heading for all your project cards.
- Your design:
![Capture3](https://user-images.githubusercontent.com/85113567/153601247-f316f7c9-2805-46c3-9e02-1d6f8163d303.PNG)
- Figma design:
![Capture4](https://user-images.githubusercontent.com/85113567/153601323-4d4fb5c9-8b51-4e80-b246-a2213378ff61.PNG)

Reply:
👍🏽 All good now.